cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -386,6 +386,14 @@ func (s *Server) updateRelease(ctx context.Context, packageRef *corev1.Installed
 		return nil, err
 	}
 
+	// per discussion will Michael 4/12/2022
+	// for now: we disallow updates to pending releases and allow them for non-pending ones
+	// (i.e. success or failed status)
```

Comment:
Rather than "per discussion with Michael ...", it'd be more useful IMO to have the reason that you presented for this change, such as "Initially we'll disallow updates to pending releases to simplify the initial case, though we may implement support later."? (I have no issue if you want to include "As Michael and I agreed, initially we'll..." if that's important for you)

Reply:
👍